docs/retry-activities-and-workflows.md

```diff
@@ -0,0 +1,185 @@
+        .setRetryOptions(
+            RetryOptions.newBuilder()
+            // Here we set the InitialInternal backoff to 1 second
+            .setInitialInterval(Duration.ofSeconds(1))
+            // Here we set the BackoffCoefficient (Temporal uses an exponential backoff)
+            .setBackoffCoefficient(2.0)
+            // Here we set the MaximumInterval backoff to 100 seconds
+            .setMaximumInterval(Duration.ofSeconds(100))
+            // Here we set the MaximumAttempts
+            // 0 is interpreted as infinite
+            // To disable Activity retries set maximumAttempts to 1
+            .setMaximumAttempts(0)
+            .build())
+        .build());
+```
+
+</TabItem>
+<TabItem value="go">
+
+```go
+ao := workflow.ActivityOptions{
+    RetryPolicy: &temporal.RetryPolicy{
+        // Here we set the InitialInternal backoff to 1 second
+		InitialInterval:    time.Second,
+        // Here we set the BackoffCoefficient (Temporal uses an exponential backoff)
+		BackoffCoefficient: 2.0,
+        // Here we set the MaximumInterval backoff to 100 seconds
+		MaximumInterval:    time.Second * 100,
+        // Here we set the MaximumAttempts
+        // 0 is interpreted as infinite
+        // To disable Activity retries set maximumAttempts to 1
+        MaximumAttempts:    0,
+	},
+}
+```
+
+</TabItem>
+</Tabs>
+
+In this second example, we are adjusting only the initial interval and doubling the coefficient.
+
+<Tabs
+    defaultValue="java"
+    values={[
+        { label: 'Java', value: 'java', },
+        { label: 'Go', value: 'go', },
+    ]
+ }>
+
+<TabItem value="java">
+
+```java
+private final RemoteDataActivities activities =
+    Workflow.newActivityStub(
+        RemoteDataActivity.class,
+        ActivityOptions.newBuilder()
+        .setScheduleToCloseTimeout(Duration.ofSeconds(10))
+        .setRetryOptions(
+            RetryOptions.newBuilder()
+            // Here we specify a custom InitialInterval
+            .setInitialInterval(Duration.ofSeconds(2))
+            // Here we specify a custom BackoffCoefficient
+            .setBackoffCoefficient(4.0)
+            .build())
+        .build());
+```
+
+</TabItem>
+<TabItem value="go">
+
+```go
+ao := workflow.ActivityOptions{
+    RetryPolicy: &temporal.RetryPolicy{
+        // Here we specify a custom InitialInternal backoff
+        InitialInterval:    time.Second * 2,
+        // Here we specify a custom BackoffCoefficient
+        BackoffCoefficient: 4.0,
+    },
+}
+ctx = workflow.WithActivityOptions(ctx, ao)
+```
+
+</TabItem>
+</Tabs>
+
+## Workflow retries
+
+While retrying Activities is enough for the majority of use cases, Temporal also provides a mechanism for retrying an entire Workflow. Workflow level retries can be very useful, such as when your Workflow consists of multiple Activities that rely on a host-specific state. Unlike Activities, Workflows are not retried by default, they must be explicitly configured to occur. 
```

Comment:
Two things:

1. link to task routing
2. they must be explicitly configured to occur. - don't like the word "occur"